### DISCUSSION TITLE
Can a script find locations of stars and output them to file

### DISCUSSION THREAD

**Discussion by emil108** [Stellarium/stellarium#1966] [Q&A]
Hi, I am interested in taking about 20 nearest stars and seeing their motion over 1000s of years showing proper motion and precession and outputting their right ascensions and declinations to file. Is this doable through a script?
Thanks for the fantastic piece of software BTW.
Emil

**Comment by gzotti** [comment]
If you know the 20 stars already, you can simply do that. Else I cannot say how to directly search for the nearest stars. Consider if you need aberration correction (rather not!). But please note Appendix F.2 in the User Guide.

**Reply by emil108** [reply]
Thank very much Georg. I found a list of nearest stars and brown dwarfs on Wikipedia.
